templates/firstboot/mellanox_fw_update.yaml

```diff
@@ -27,11 +27,11 @@ parameters:
     description: 'Link type ["eth", "ib"]'
   ESWITCH_IPV4_TTL_MODIFY_ENABLE:
     type: boolean
-    default: True
+    default: False
```

Comment:
why was this changed?

Reply:
So the default should be False, but we didn't rebase the changes on my local repo to this Mellanox repo